.prettierrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
       files: '*.{js,ts}',
       options: {
         singleQuote: true,
+        trailingComma: 'es5',
```

Comment:
Wait, I can't remember ever seeing that after an update. Is that something you added yourself?

Reply:
I can do that, but it started expecting a trailingComma everywhere which resulted in +40 files changed. Which seemed pretty excessive. 

Also I thought we used es5 implicitly for most frontends